src/Catalog/NuGet.Services.Metadata.Catalog.csproj

```diff
@@ -94,20 +94,17 @@
       <HintPath>..\..\packages\Newtonsoft.Json.9.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="NuGet.Common, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\NuGet.Common.4.0.0\lib\net45\NuGet.Common.dll</HintPath>
+    <Reference Include="NuGet.Common, Version=4.3.0.3, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\NuGet.Common.4.3.0-preview3-4112\lib\net45\NuGet.Common.dll</HintPath>
```

Comment:
I recommend taking 4146 as it is the most recent build we've inserted into Visual Studio internally.  It's better than an arbitrary dev branch daily build.

This feedback applies throughout the PR.